cluster/examples/kubernetes/ceph/csi/example/rbd/configure-storageclass.bash

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+set -eux
+
+######################################
+# Example helper script to automate:
+# 1. Creating a CephBlockPool
+# 2. Verify existence of a kuberentes login on the Ceph cluster
+# 3. Fetches base64 encoded keys for admin and kubernetes logins on cluster and creates a Secret object with them
+# 4. Parses out the Cluster IPs for the Ceph mons  and creates a csi-rbd storageclass
+######################################
+
+<< EOF | kubectl create -f -
```

Comment:
cat <<EOF | kubectl create -f -

Reply:
Ha!  Funny enough that's what I originally had and I was asked to remove the "cat" because it was viewed as "redundant"